### DISCUSSION TITLE
how to calculate the max ROR and flag it on the BT curve

### DISCUSSION THREAD

**Discussion by mansonchan888** [artisan-roaster-scope/artisan#1135] [Q&A]
1) how to calculate flag / alarm with max ROR occur, it sound odds to mark this event but
I am trying to use it to learn more about artisan alarm, event, button and symbolic variable by using this example
If you know know how to get there, please share with me

<img width="531" alt="WeChat3fb93fd9c91bbefbed7b4830b5b4e854" src="https://user-images.githubusercontent.com/119805088/226491527-0561f734-b7f1-42ec-b381-87313d159c6a.png">

**Comment by poundy** [comment]
Do you think that this is sensible? At some point, the RoR might well be above what you "calculate" it at, for example a temp sample that almost hit the crest but has two "devils horns" as peaks (or your probes might just have a wide range of variability). But I'd go further and ask what good does having that marked as an event do you when you're one minute past that event anyway?  (I know the original source of that diagram, and if you were at that training you'd know that not much happens explicitly around that event.) Plus you can, if you were so inclined, go mark an event later once you have the roast done or once you have quiet time, by looking at the RoR line (it's pretty easy to see).
